exist-core/src/test/xquery/xquery3/serialize.xql

```diff
@@ -243,7 +243,7 @@ function ser:adaptive-simple-atomic($atomic as xs:anyAtomicType) {
 
 declare
     %test:args(1234)
-    %test:assertEquals('"1234"')
+    %test:assertEquals(1234)
```

Comment:
see above

Reply:
As above - this is not related to adaptive serialization.

Reply:
We have to be able to test that a string is returned. If this happens to be castable to an integer value should not affect the assertion.